pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -55,7 +55,7 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		return "", errors.Wrapf(err, "failed to set owner reference to mon service %q", svcDef.Name)
 	}
 
-	// If deploying Nautilus or newer we need a new port for the monitor service
+	// If deploying Octopus or newer we need a new port for the monitor service
```

Comment:
IMO, it is still most correct to leave this as "Nautilus". It's a code comment so doesn't affect runtime.